### DISCUSSION TITLE
Reference layout for keyboard-layout-editor.com

### DISCUSSION THREAD

**Discussion by gehel** [sevmeyer/chrumm-keyboard#6] [Show and tell]
This isn't really an issue, but might be interesting to someone. I've built a Chrumm (amazing project! Thanks!) and it was useful to me to have a reference keyboard. I've used keyboard-layout-editor.com to prepare my layout. I'm using Bépo, which might not be everyone's cup of tea, but that might be a good starting point for someone else. So here it is: http://www.keyboard-layout-editor.com/#/gists/4b3e1e1e5b09e7161a9d690c0c770ef8

**Comment by sevmeyer** [comment]
Neat, thank you! I have moved this thread to the "Discussions" tab, to keep things organized.
